tests/jerry/es.next/module-resource-name.js

```diff
@@ -14,14 +14,19 @@
 
 import { getName, getNamePromise } from "./module-resource-name-export.mjs"
 
-assert(getName().endsWith("/module-resource-name-export.mjs") === true);
+function modulePathEndsWith(path, name)
+{
+  return path.endsWith('/' + name) || path.endsWith('\\' + name)
+}
+
+assert(modulePathEndsWith(getName(), "module-resource-name-export.mjs") === true);
```

Comment:
I think that this is an overcomplication. Wouldn't it be simply enough to remove the slashes from the paths? E.g.
`assert(getName().endsWith("module-resource-name-export.mjs") === true);`

BTW, `String.prototype.endsWith` returns a boolean. Is it necessary to compary it to `true` in an assert? I think that the `=== true` part could be dropped, that would be more natural.